app/include/dt-bindings/zmk/hid_usage_pages.h

```diff
@@ -48,4 +48,5 @@
 #define HID_USAGE_CAMERA (0x90)         // Camera Control
 #define HID_USAGE_ARCADE (0x91)         // Arcade
 #define HID_USAGE_GAMING (0x92)         // Gaming Device
-#define HID_USAGE_FIDO (0xF1D0)         // FIDO Alliance
\ No newline at end of file
+#define HID_USAGE_FIDO (0xF1D0)         // FIDO Alliance
+#define HID_USAGE_PLOVER (0xFF50)       // Plover HID vendor-defined usage page
```

Comment:
I'd suggest a slight naming tweak, to be clear about how "official" this is.


```suggestion
#define HID_USAGE_VENDOR_PLOVER (0xFF50)       // Plover HID vendor-defined usage page
```

Reply:
Yes, very good suggestion, I'll do that.